dockernet/config.sh

```diff
@@ -39,6 +39,9 @@ elif [[ "${#HOST_CHAINS[@]}" == "0" ]]; then
 fi
 REWARD_CONVERTER_HOST_ZONE=${HOST_CHAINS[0]}
 
+# TODO[airdrop]: revert before merging
```

Comment:
@sampocs the pr is full of these kind of things. brabched off airdrop-module and merged from main. 🤷‍♂️